pkgs/development/compilers/rust/rustc.nix

```diff
@@ -227,6 +227,13 @@ in stdenv.mkDerivation (finalAttrs: {
 
     # Useful debugging parameter
     # export VERBOSE=1
+  '' + lib.optionalString stdenv.targetPlatform.isDarwin ''
```

Comment:
I think this actually needs to be `stdenv.hostPlatform.isDarwin || stdenv.targetPlatform.isDarwin`, because a Darwin rustc targeting Linux would still need this changed too.

Reply:
Done.